api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -574,7 +565,7 @@ public static Module getModuleByPackage(String modulePackage) {
 	 * @see Daemon#startModule(Module)
 	 */
 	public static Module startModule(Module module) throws ModuleException {
-		return Daemon.startModule(module);
```

Comment:
I will look at the examples you proposed in ModuleActivatorTest and see if can get the tests working before the end of tomorrow :).